modules/deployment/src/main/resources/terraformbootApi.json

```diff
@@ -35,8 +35,8 @@
           "required": true
         },
         "responses": {
-          "400": {
```

Comment:
it seems only the order of some contents has changed in this file, can you remove this from this PR?

Reply:
This file was copied from the swagger api-docs of the terraform-boot. If the terraform boot changes, it must also be changed here.